pkg/reader/ccip.go

```diff
@@ -552,6 +552,7 @@ func (r *ccipChainReader) LatestMsgSeqNum(
 		return 0, fmt.Errorf("message invalid msg %v: %w", msg, err)
 	}
 
+	lggr.Debugw("chain reader returning latest message sequence number", "seqNum", msg.Message.Header.SequenceNumber)
```

Comment:
There's a `queried latest message from source` log above that is `Infow` but probably should be `Debugw`. Maybe this can be `Infow`?